spec/requests/posts_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'PostsController', type: :request do
+  describe 'GET /index' do
+    context 'when the page is loaded' do
+      it 'returns a 200 status code and render template for index' do
+        get '/users/1/posts'
+        expect(response).to have_http_status(200)
+        expect(response).to render_template(:index)
+      end
+      it 'includes pagination' do
+        get '/users/1/posts'
+        expect(response.body).to include('Pagination')
+      end
```

Comment:
- [ ] This test failed. There is no `Pagination` in your post index template. Kindly fix it.
<img width="319" alt="Code_dihbYyH4sB" src="https://user-images.githubusercontent.com/81468675/179757437-0996efe6-6658-477c-baf7-e34d2a2c1a54.png">